pygeoapi/flask_app.py

```diff
@@ -155,6 +155,36 @@ def conformance():
     return get_response(api_.conformance(request))
 
 
+@BLUEPRINT.route('/TileMatrixSets/WorldCRS84Quad')
```

Comment:
Can we manage the route parameter dynamically like `@BLUEPRINT.route('/TileMatrixSets/<TileMatrixSetId>')`?